README.md

```diff
@@ -109,3 +109,35 @@ You've just successfully set up the environment for the gDR umbrella package. Pl
 
 # How to contribute
 We are eager to make this tool useful for the community. We welcome any suggestions via GitHub issues, or pull requests for bugfixes/new features. When making pull requests, please first have a look at the style conventions in [gDRstyle](https://gdrplatform.github.io/gDRstyle/style_guide.html).
+
+# Preparing your machine to run gDR images
+1. How to install Docker.
```

Comment:
Maybe `1. Install Docker` and below add `How to install Docker` with list in bulletpoints?